frontend/src/client/models.ts

```diff
@@ -34,6 +34,7 @@ export type ItemsPublic = {
 };
 
 export interface InquiryCreate {
+  id: string;
```

Comment:
I don't think you want an id attribute on InquiryCreate.  An Inquiry doesn't have an id until after it is created.  Can you please check if this is necessary.

Reply:
I put an id because I got an error that it was missing. So my guess is it's required. Doesn't hurt.

Reply:
The problem is you were reusing this definition for updates.You need a separate typedef for inquiryUpdate. That has an id.  inquiryCreate does not have an id.